core/src/main/java/hudson/Util.java

```diff
@@ -1227,21 +1227,53 @@ public static String fixEmptyAndTrim(@CheckForNull String s) {
         return fixEmpty(s.trim());
     }
 
+    /**
+     *
+     * @param l list to check
+     * @param <T>
```

Comment:
empty Javadoc params cause warnings in Javadoc linter. It would be better to put some text here

Reply:
of course.

Reply:
I have added "Type of the list." just below.
Just like https://github.com/jenkinsci/jenkins/blob/22aa2e6e766074d11249893e3f35e0b99e20d3d0/core/src/main/java/jenkins/ExtensionComponentSet.java#L52

Reply:
Sorry, missed that